changelogs/fragments/3044-proxmox-inventory-snapshots.yml

```diff
@@ -0,0 +1,2 @@
+new_features:
+  - proxmox inventory - added snapshots to host facts (https://github.com/ansible-collections/community.general/pull/3044)
```

Comment:
```suggestion
minor_changes:
  - proxmox inventory plugin - added snapshots to host facts (https://github.com/ansible-collections/community.general/pull/3044).
```

Reply:
Done!

Reply:
Can you add a `.` to the end of the sentence as well? After that this is good to go, thanks!